app/services/update_user.rb

```diff
@@ -0,0 +1,14 @@
+class UpdateUser
```

Comment:
I don't mind this abstraction. 

Agree that it would be easier to read if this update happened directly in the form object, but I like the idea of future proofing / removing dependencies in specs. 

Maybe we can add a ticket somewhere for a refactor task to use this class everywhere that we update users (for consistency)?